src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -1184,7 +1184,7 @@ void M()
         DateTime dt = System.DateTime.MaxValue$$
     }
 }",
-                MainDescription($"({FeaturesResources.field}) DateTime DateTime.MaxValue"));
+                MainDescription($"(static readonly {FeaturesResources.field}) DateTime DateTime.MaxValue"));
```

Comment:
is this done for properties as well?

Reply:
Per issue [#26947](https://github.com/dotnet/roslyn/issues/26947), properties are not in scope and would require a separate design. If we like this for fields, I think it could make sense to expand to other member types as well